go/install/fuse_status_windows.go

```diff
@@ -6,33 +6,73 @@
 package install
 
 import (
+	"errors"
 	"path/filepath"
 
+	"github.com/gonutz/w32"
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/protocol/keybase1"
 )
 
-func detectDokanDll(log Log) bool {
-	dir, err := libkb.SystemDir()
-	if err != nil {
-		log.Info("detectDokanDll error getting system directory: %v", err)
-		return false
+func isDokanCurrent(log Log, path string) (bool, error) {
+	size := w32.GetFileVersionInfoSize(path)
+	if size <= 0 {
+		return false, errors.New("GetFileVersionInfoSize failed")
+	}
+
+	info := make([]byte, size)
+	ok := w32.GetFileVersionInfo(path, info)
+	if !ok {
+		return false, errors.New("GetFileVersionInfo failed")
 	}
 
-	exists, _ := libkb.FileExists(filepath.Join(dir, "dokan1.dll"))
+	fixed, ok := w32.VerQueryValueRoot(info)
+	if !ok {
+		return false, errors.New("VerQueryValueRoot failed")
+	}
+	version := fixed.FileVersion()
+
+	major := version & 0xFFFF000000000000 >> 48
+	minor := version & 0x0000FFFF00000000 >> 32
+	patch := version & 0x00000000FFFF0000 >> 16
+	build := version & 0x000000000000FFFF >> 0
+
+	// we're looking for 1.1.0.1000
+	result := major >= 1 && minor >= 1 && build >= 1000
```

Comment:
This check will fail `1.1.1.0` because of `build >= 1000`. I think what you want is:

```
result := major > 1 || (
    major == 1 && (
        minor > 1 || (
            minor == 1 && (
                patch > 0 || (
                    patch == 0 && build >= 1000)))))
```

Reply:
Ha, whoops! sorry about that